packages/core/src/Select/Option.tsx

```diff
@@ -0,0 +1,72 @@
+import { useRef } from "react";
+import { useOption } from "@mui/base/useOption";
+import { OptionOwnProps } from "@mui/base/Option";
+import { useForkRef } from "@mui/material/utils";
+
+import { HvListItem, HvListItemProps } from "../ListContainer";
+import { useDefaultProps } from "../hooks/useDefaultProps";
+import { fixedForwardRef } from "../types/generic";
+import { ExtractNames, createClasses } from "../utils/classes";
+import { outlineStyles } from "../utils/focusUtils";
+
+const { staticClasses, useClasses } = createClasses("HvOption", {
+  root: {},
+  highlighted: {
+    ...outlineStyles,
+  },
+});
+
+export { staticClasses as optionClasses };
+
+export type HvOptionClasses = ExtractNames<typeof useClasses>;
+
+export interface HvOptionProps<OptionValue extends {}>
+  extends Omit<HvListItemProps, "value" | "disabled">,
+    Pick<OptionOwnProps<OptionValue>, "disabled" | "label" | "value"> {
+  classes?: HvOptionClasses;
+}
+
+export const HvOption = fixedForwardRef(function HvOption<
```

Comment:
I was wondering if it wouldn't be better to use `HvSelectOption` instead of `HvOption` to keep the more generic `HvOption` name free in case we need it in the future. Same thing for the `HvOptionGroup`. Not sure about this one, it was just a thought.

Reply:
I thought of that too. I ended up going with `HvOption` because it's shorter and it maps _directly_ to the HTML element `<option>`. Similarly for `HvOptionGroup` and `<optgroup>` (maybe we could call it `HvOptGroup` instead) 🤔

But I don't have a strong opinion on this.